src/integer_mod_q/poly_over_zq/from.rs

```diff
@@ -48,6 +50,47 @@ impl From<&Modulus> for PolyOverZq {
     }
 }
 
+impl From<&Zq> for PolyOverZq {
+    /// Creates a constant [`PolyOverZq`] with a specified constant.
+    ///
+    /// # Parameters:
+    /// - `value`: the constant value the polynomial will have.
+    ///   It has to be a [`Zq`], or a value that can be converted into [`Zq`].
+    ///   
+    /// Returns a new constant polynomial with the specified value and modulus.
+    ///
+    /// # Examples:
+    /// ```
+    /// use qfall_math::{integer_mod_q::*, traits::*};
+    ///
+    /// let poly = PolyOverZq::from((1, 10));
+    /// let poly_2 = PolyOverZq::from(Zq::from((1, 10)));
+    ///
+    /// let value_cmp: Zq = poly.get_coeff(0).unwrap();
+    /// assert_eq!(value_cmp, Zq::from((1, 10)));
+    /// assert_eq!(poly.get_degree(), 0);
+    /// assert_eq!(poly, poly_2);
+    /// ```
```

Comment:
```suggestion
    /// ```
    /// use qfall_math::{integer_mod_q::*, traits::*};
    /// let poly = PolyOverZq::from((1, 10));
    ///
    /// let poly_2 = PolyOverZq::from(Zq::from((1, 10)));
    ///
    /// assert_eq!(poly, poly_2);
    /// assert_eq!(poly.get_degree(), 0);
    /// ```
```
This example should be more intuitive and provide more insight into *this* functions i/o.

Reply:
I used the new example, but in the format of the other examples.